### PR TITLE
Adds event tracking for non-button clicks to homepage websites.

### PR DESCRIPTION
**Pull request by chuckharmston**
r? @ngokevin

### CODE REVIEW

File: src/templates/_macros/feed_websites.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
             <button class="feed-item-website-tabs-tab">
               {{ deferred_icon(website.icons['64'], name) }}
             </button>
-            <a href="{{ website.url }}" class="feed-item-website-tabs-content mkt-website-link">
+            <a href="{{ website.url }}" class="feed-item-website-tabs-content mkt-website-link" data-product="{{ website|json }}">
```
Comment (ngokevin) [inline src/templates/_macros/feed_websites.html]: hahaha

File: src/templates/_macros/feed_websites.html [hunk 2]
```diff
@@ -31,7 +31,7 @@
         {% set website = apps.transform(website) %}
         {% set name = website.name|translate(website) %}
         <li>
-          <a class="mkt-tile mkt-website-link" data-slug="{{ website.slug }}" data-id="{{ website.id }}" href="{{ website.url }}" data-content-type="{{ website.contentType }}">
+          <a class="mkt-tile mkt-website-link" data-slug="{{ website.slug }}" data-id="{{ website.id }}" href="{{ website.url }}" data-content-type="{{ website.contentType }}" data-product="{{ website|json }}">
```
Comment (ngokevin) [inline src/templates/_macros/feed_websites.html]: we can line break when these lines get too long

**Comment by ngokevin** [comment]
r+

**Comment by chuckharmston** [comment]
Thanks!

**Pull request merged**